### ISSUE TITLE
Add moment locales for Tagalog (tl) Illonggo (hil) and Bisaya (ceb) languages

### ISSUE THREAD

**Issue by dianabarsan** [medic/cht-core#6861]
**Is your feature request related to a problem? Please describe.**
A new app in the Philippines is requring language support for Tagalog, Illonggo, and Bisaya. In addition to #6848 (blocking), the project will also require translations for the moment library. 

**Describe the solution you'd like**
Tagalog already has a locale in the moment library, called `tl-ph` (to note that the language code needs to match! so we'd need to use `tl-ph` language code for Tagalog. (https://github.com/moment/moment/blob/develop/locale/tl-ph.js)

Illonggo and Bisaya moment locales need to be created and added to cht-core code (use existent moment locale files to see what the structure looks like).

**Comment by dianabarsan** [comment]
cc @craig-landry @binokaryg

**Comment by dianabarsan** [comment]
As an addendum, if the locale code doesn't exist, moment will default to english (not to a "fallback" locale of choice). 
https://github.com/medic/cht-core/blob/master/webapp/src/js/services/language.js#L35

**Comment by binokaryg** [comment]
>  we'd need to use tl-ph

Everywhere? Including #6849?

@joymkimmel please provide the translations for these too.

**Comment by dianabarsan** [comment]
Yes, the language code to use needs to be `tl-ph` to use the moment locale. 
If we don't want to use `tl-ph`, then we'd need to create a moment locale file for Tagalog as well (we'd just copy the contents from the `tl-ph` translation from the moment repo) and use it just like the other two.

**Comment by kennsippell** [comment]
@dianabarsan I looked for use of `moment.format` in cht-core and found lots of references in API, and admin console, but couldn't find relevant uses in webapp. It was a quick search, so I probably messed that up. But just confirming that moment's translation features are still relevant to the end-user experience?

**Comment by dianabarsan** [comment]
We use moment's `relativeTime` and `pastFuture` localeData when displaying a lot of relative times within the app (think even filters that are used in contact summary):
https://github.com/medic/cht-core/blob/b747f4731b6eec602fae2ab0aa1ceb6aa0074f3e/admin/src/js/services/format-date.js#L86

More info about `moment.localeData` here: https://momentjs.com/docs/#/i18n/locale-data/

**Comment by dianabarsan** [comment]
We also use `moment.format` specifically here: https://github.com/medic/cht-core/blob/b747f4731b6eec602fae2ab0aa1ceb6aa0074f3e/webapp/src/js/services/format-date.js#L42

**Comment by joymkimmel** [comment]
hi guys, sorry for the delay. I have the 3 moment files I had to change it to .txt for it to upload here. If you want me to put the js files somewhere else let me know. It would be good I think for someone to double check the formatting. I changed all of the weeks to start on Sunday (0). 

[ceb-ph.txt](https://github.com/medic/cht-core/files/5847730/ceb-ph.txt)
[hil-ph.txt](https://github.com/medic/cht-core/files/5847731/hil-ph.txt)
[tl-ph.txt](https://github.com/medic/cht-core/files/5847732/tl-ph.txt)

**Comment by dianabarsan** [comment]
Thank you @joymkimmel 

@binokaryg it's possible that adding these files with this exact formatting to our repo could trigger linting errors. Do you think you're able to fix those yourself or would you like me to take this task on?

**Comment by dianabarsan** [comment]
Also, @binokaryg , please confirm what language codes you intended to use for the 3 languages. We will have to name these files exactly like the languages codes you will be choosing.

**Comment by binokaryg** [comment]
I will try to fix them according to the `eslint` rules of cht-core. Else, there's always `/* eslint-disable */`.

For the languages, I think we can keep:
Tagalog: `tl`
Illonggo: `hil`
Bisaya: `ceb`
@joymkimmel Agree?

**Comment by dianabarsan** [comment]
ok @binokaryg . My suggestion is to completely remove the `factory` block that wraps the locale definition and instead just do:
```
const moment = require('moment');
moment.defineLocale(<code>, <locale def>);
```

**Comment by joymkimmel** [comment]
@binokaryg yes, agreed.

**Comment by dianabarsan** [comment]
Ready for at on `binokaryg:master`.

**Comment by ngaruko** [comment]
LGTM.
Tested on @binokaryg's [repo](https://github.com/binokaryg/cht-core). Feel free to merge @dianabarsan or @craig-landry . I/QA will do more overall release testing  around [languages and locale](https://github.com/medic/medic-release-testing/issues/113)s once a beta is available.
![image](https://user-images.githubusercontent.com/6979995/105427089-3f8b6980-5cb1-11eb-8daa-2256e22d32a2.png)

**Comment by dianabarsan** [comment]
Merged to `master` and cherry picked to `3.10.x`
